### ISSUE TITLE
Accessibility: Avatar selection

### ISSUE THREAD

**Issue by etoledom** [Automattic/Gravatar-SDK-iOS#673]
With VoiceOver turned on:

When selecting an avatar, screen reader will only read "image".

Expected:

When selecting an avatar, screen reader should read something like:
**Not selected:** "Avatar image, - alt description -, button. Double tap to select. 
**Selected:** "Avatar image, selected. - Alt description - "

- When changing the label after selecting the image, VoiceOver should read it again, now with the "selected" word added to it. If this doesn't occur. There should be an implementation to make this happen.

https://github.com/user-attachments/assets/cd73ae16-3780-4b15-9ec6-bdde118a8cd3
